src/docs/projects/staticanalysis-frontend.rst

```diff
@@ -0,0 +1,22 @@
+.. _staticanalysis/frontend-project:
+
+Project: staticanalysis/frontend
+===============================
+
+:contact: `Bastien Abadie`_, (backup `Release Management`_)
+
+
+0. Run the frontend
+"""""""""""""""""""
+
+You can start the frontend with this command:
+
+.. code-block:: shell
+
+  ./please run staticanalysis/frontend
+
+The you can then access the frontend at https://127.0.0.1:8010/#/ (note: you may habe to bypass a warning about the self-signed HTTPS certificate).
```

Comment:
Nit: a couple of spelling mistakes here: `The you can then access` and `you may habe to`.

Reply:
Oops, will fix, thanks for catching them!